config/config_test.go

```diff
@@ -773,6 +773,7 @@ func TestLocal_ValidateP2PHybridConfig(t *testing.T) {
 
 	for i, test := range tests {
 		test := test
+		i := i
```

Comment:
Was lint complaining? I don't see why, unless there's code taking the address of `i` which would be unusual.